### ISSUE TITLE
Component lifecycle: update input mask

### ISSUE THREAD

**Issue by annepetersen** [uswds/uswds-site#3185]
### Describe the bug

Input mask has failed WCAG 2.1 AA: this is reflected appropriately on the [main component page for input mask](https://designsystem.digital.gov/components/input-mask/), the [input mask known issue section](https://designsystem.digital.gov/components/input-mask/#known-issues) and [input mask's accessibility test page](https://designsystem.digital.gov/components/input-mask/accessibility-tests/), but not the [component status page](https://designsystem.digital.gov/components/status/). 

On the component status page, input mask's entry should read "Use with caution," reflecting the appropriate [lifecycle phase](https://designsystem.digital.gov/components/lifecycle/#use-with-caution) for the current version of input mask.

### Steps to reproduce the bug

1. Go to https://designsystem.digital.gov/components/status/
2. Scroll down to Input mask and read the "Released" column label. It should read "Use with caution" and be in a different color than "Released" labels.

### Expected Behavior

Input mask's "Released" column label should read "Use with caution" and be in a different color than "Released" labels. The appropriate color to use has not yet been established since this is the first instance of an accessibility failure of a USWDS component that we have not been able to address promptly (to my knowledge).

### Screenshots

Current status page:
<img width="693" height="684" alt="Image" src="https://github.com/user-attachments/assets/3b2c24da-c124-4449-9f5c-af3d59940bf1" />

From the input mask component page:
<img width="770" height="230" alt="Image" src="https://github.com/user-attachments/assets/387f84d2-9a80-4b4b-a0ef-aa2ac99032ca" />

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/uswds-site/issues) for duplicate bug reports.
